index.html

```diff
@@ -0,0 +1,89 @@
+</body>
+</html>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <link rel="stylesheet" href="styles.css">
+    <title>YouTube Replica</title>
+</head>
+<body>
+
+    <header class="header">
```

Comment:
I think :thinking: the use of class names similar to HTML tags can cause confusion, especially when working with CSS.Try using unique class names.:blush: